doc/stdenv/meta.xml

```diff
@@ -190,7 +190,7 @@ hello-2.3  A program that produces a familiar, friendly greeting
     <listitem>
      <para>
       A list of names and e-mail addresses of the maintainers of this Nix expression. If you would like to be a maintainer of a package, you may want to add yourself to <link
-    xlink:href="https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix"><filename>nixpkgs/maintainers/maintainer-list.nix</filename></link> and write something like <literal>[ stdenv.lib.maintainers.alice stdenv.lib.maintainers.bob ]</literal>.
+    xlink:href="https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix"><filename>nixpkgs/maintainers/maintainer-list.nix</filename></link>  (in a separate commit titled 'maintainers: add alice'), and write something like <literal>[ stdenv.lib.maintainers.alice stdenv.lib.maintainers.bob ]</literal>.
```

Comment:
Maybe we can also use `with` in the example like most of the packages do these days?

```suggestion
    xlink:href="https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix"><filename>nixpkgs/maintainers/maintainer-list.nix</filename></link>  (in a separate commit titled 'maintainers: add alice'), and write something like <literal>with stdenv.lib.maintainers; [ alice bob ]</literal>.
```